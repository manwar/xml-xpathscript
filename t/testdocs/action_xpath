<doc>
<a href="/some/where"><blah></blah></a>
<a href="/some/where"><blah></blah></a>
</doc>
