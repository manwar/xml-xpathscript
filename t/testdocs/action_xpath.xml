<doc>
    <foo>
        <a />
        <b process="yes" />
        <c/>
        <d process="yes" />
    </foo>
</doc>
