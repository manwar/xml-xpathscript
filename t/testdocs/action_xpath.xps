<%
$template->set( foo => { action => './*[@process = "yes"]' } );
%><%= apply_templates() %>
