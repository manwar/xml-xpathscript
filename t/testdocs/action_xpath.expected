<doc>
    <b process="yes"></b><d process="yes"></d>
</doc>
